deeplake/core/tensor.py

```diff
@@ -1734,36 +1780,41 @@ def create_vdb_index(
         additional_params: Optional[Dict[str, int]] = None,
     ):
         """
-        Create similarity search index for embedding tensor.
+        Create similarity search index for embedding tensor or inverted index for text tensor.
 
         Args:
-            id (str): Unique identifier for the index. Defaults to ``hnsw_1``.
+            id (str): Unique identifier for the index. Defaults to ``hnsw_1``. or ``inverted_index1``.
             distance (DistanceType, str): Distance metric to be used for similarity search. Possible values are "l2_norm", "cosine_similarity". Defaults to ``DistanceType.COSINE_SIMILARITY``.
             additional_params (Optional[Dict[str, int]]): Additional parameters for the index.
-                - Structure of additional params is:
+                - Structure of additional params is used for HNSW index:
                     :"M": Increasing this value will increase the index build time and memory usage but will improve the search accuracy. Defaults to ``16``.
                     :"efConstruction": Defaults to ``200``.
                     :"partitions": If tensors contain more than 45M samples, it is recommended to use partitions to create the index. Defaults to ``1``.
+                - Structure of additional params is used for Inverted index:
+                    :"bloom_filter_size": Size of the bloom filter. Defaults to ``100000``.
+                    :"segment_size": Size of the segment in MB. Defaults to ``25``.
 
         Example:
             >>> ds = deeplake.load("./test/my_embedding_ds")
             >>> # create cosine_similarity index on embedding tensor
             >>> ds.embedding.create_vdb_index(id="hnsw_1", distance=DistanceType.COSINE_SIMILARITY)
             >>> # create cosine_similarity index on embedding tensor with additional params
             >>> ds.embedding.create_vdb_index(id="hnsw_1", distance=DistanceType.COSINE_SIMILARITY, additional_params={"M": 32, "partitions": 1, 'efConstruction': 200})
+            >>> # create inverted index on text tensor
+            >>> ds.text.create_vdb_index(id="inverted_index1")
+            >>> # create inverted index on text tensor with additional params
+            >>> ds.text.create_vdb_index(id="inverted_index1", additional_params={"bloom_filter_size": 1000000, "segment_size": 50})
 
         Notes:
-            Index creation is supported only for embedding tensors.
+            Index creation is supported only for embedding tensors and text tensors.
```

Comment:
`Index creation is supported for embedding tensors and text tensors.`

Reply:
Done.